workflow/snakemake_rules/export.smk

```diff
@@ -0,0 +1,54 @@
+build_dir = config.get("build_dir", "builds")
+
+def _get_node_data_by_wildcards(wildcards):
+    """Return a list of node data files to include for a given build's wildcards.
+    """
+    # Define inputs shared by all builds.
+    wildcards_dict = dict(wildcards)
+    inputs = [
+        rules.refine.output.node_data,
+        rules.ancestral.output.node_data,
+        rules.translate.output.node_data,
+        rules.clades.output.node_data,
+        rules.traits.output.node_data
+    ]
+
+    if config.get('titer-models',False):
+        inputs.append(rules.titers_sub.output.titers_model)
+        inputs.append(rules.titers_tree.output.titers_model)
+    if config.get('glycosilation', False) and wildcards.segment in ['ha', 'na']:
+        inputs.append(rules.glyc.output.glyc)
+    if config.get('lbi', False) and wildcards.segment in ['ha', 'na']:
+        inputs.append(rules.lbi.output.lbi)
```

Comment:
These top-level boolean keys like `titer-models` don't allow us to limit which segments to apply the models to from the config, so we'll run titer models on NA, PB1, etc. Could we move these types of parameters into the build definitions (like the `vaccines` option below) instead so we have more control over which builds use which annotations?

Then we can eliminate all of these additional segment tests that happen for glycosylation, LBI, etc.